packages/calypso-build/package.json

```diff
@@ -35,22 +35,11 @@
 	],
 	"types": "./typings/calypso-assets/index.d.ts",
 	"dependencies": {
-		"@automattic/babel-plugin-preserve-i18n": "workspace:^",
```

Comment:
Do we need a dependency on `@automattic/calyps-babel-config` since we utilize it in webpack?

Reply:
We totally do. Good catch!